### PR TITLE
Making Crypto/Defi/DPI --sort Pretty Again

### PR DESCRIPTION
**Pull request by simmonsj330**
# Description
Making changes to revert back to the use of "M" (millions), "B" (billions), and "%" (percent), which looks better than simply using floats.

Screen Shot 2022-10-19 at 10 50 55 PM

Screen Shot 2022-10-19 at 10 50 43 PM

# How has this been tested?
Re-ran pytest on the model and view.

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: openbb_terminal/cryptocurrency/defi/defipulse_model.py [hunk 1]
```diff
@@ -12,6 +12,23 @@
...
 
 
+@log_start_end(log=logger)
+def num_to_int(num):
+    if num[-1] == "M":
+        return int(float(num[1:-1])) * 1000000
+    if num[-1] == "B":
+        return int(float(num[1:-1])) * 1000000000
+    if num[-1] == "k":
+        return int(float(num[1:-1])) * 1000
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/defi/defipulse_model.py:22]: small change.  Probably want to make sure that we look at .upper() and match the "k" to "K".

File: openbb_terminal/cryptocurrency/defi/defipulse_model.py [hunk 2]
```diff
@@ -12,6 +12,23 @@
 logger = logging.getLogger(__name__)
 
 
+@log_start_end(log=logger)
```
Comment (DidierRLopes) [inline openbb_terminal/cryptocurrency/defi/defipulse_model.py:15]: I don't think we want to log a helper func

File: openbb_terminal/cryptocurrency/defi/defipulse_model.py [hunk 3]
```diff
@@ -12,6 +12,23 @@
...
+def num_to_int(num):
```
Comment (DidierRLopes) [inline openbb_terminal/cryptocurrency/defi/defipulse_model.py:16]: type hint missing and doctoring

File: openbb_terminal/cryptocurrency/defi/defipulse_model.py [hunk 4]
```diff
@@ -12,6 +12,23 @@
...
+    if num[-1] == "M":
+        return int(float(num[1:-1])) * 1000000
```
Comment (DidierRLopes) [inline openbb_terminal/cryptocurrency/defi/defipulse_model.py:18]: For visibility, you can do: `1_000_000` which is quite cool.

Also, int(float(X)) is redundant, you can just do `int`

File: openbb_terminal/cryptocurrency/defi/defipulse_model.py (hunk 3 above)
Comment (DidierRLopes) [reply openbb_terminal/cryptocurrency/defi/defipulse_model.py:16]: Seems like the "num" var is a bit confusing, as you are playing with a string.

File: openbb_terminal/cryptocurrency/defi/defipulse_model.py [hunk 5]
```diff
@@ -12,6 +12,23 @@
...
+    if num[-1] == "B":
+        return int(float(num[1:-1])) * 1000000000
+    if num[-1] == "k":
+        return int(float(num[1:-1])) * 1000
+
+    return int(num)
+
+
+@log_start_end(log=logger)
```
Comment (DidierRLopes) [inline openbb_terminal/cryptocurrency/defi/defipulse_model.py:27]: Same comments as above apply.

I would also avoid creating a function with 1 line of code. I would look into lambda to do that one liner function: 

e.g. `df['age']=df.apply(lambda x: x['age']+3,axis=1)` See https://www.analyticsvidhya.com/blog/2020/03/what-are-lambda-functions-in-python/

**Review by DidierRLopes — CHANGES_REQUESTED** [review]

File: openbb_terminal/cryptocurrency/defi/defipulse_model.py (hunk 1 above)
Comment (simmonsj330) [reply openbb_terminal/cryptocurrency/defi/defipulse_model.py:22]: I thought thousand was normally lowercase?

**Comment by simmonsj330** [comment]
Closing because command was removed

**Pull request closed without merging**